core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -6888,6 +6891,31 @@ static class ExtendedExpander extends Expander {
 
       return super.visit(literal);
     }
+
+    /**
+     * Returns whether a given node contains a {@link SqlIdentifier}.
+     *
+     * @param parent a SqlNode
+     * @param children a SqlIdentifier
+     */
+    private boolean containsIdentifier(SqlNode parent, SqlIdentifier children) {
```

Comment:
There is no need to use plural for second parameter. Further more, we even do not need to name them 'parent, child', it's not helpful for readability in my opinion.   
How about name it `containsIdentifier(SqlNode sqlNode, SqlIdentifier target)`?

Reply:
Thanks, I renamed them.